data/hai/hai outfits.txt

```diff
@@ -812,6 +826,33 @@ outfit "Pulse Rifle"
 	"unplunderable" 1
 	description "The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to more effectively incapacitate its target, if several bolts land in quick succession, the accumulated charge has a lethal effect on most species' nervous systems."
 
+outfit "Dual Pulse Pistols"
+	category "Hand to Hand"
+	plural "Dual Pulse Pistols"
+	cost 77777
+	thumbnail "outfit/dual pulse pistols"
+	"capture attack" 2.0
+	"capture defense" 1.4
+	"unplunderable" 1
+	description "A pair of modified heavy pulse pistols that are favored by some Unfettered and the occasional pirate captain. Dual wielding them adds more firepower during short raids against a ship's defenders. Individually, they lose their energy charge more quickly than a pulse rifle, making them a poor choice for holding a defensive position during a long siege."
+
+
+outfit "Unfettered License"
+	category "Licenses"
+	thumbnail "outfit/true unfettered license"
+	"unplunderable" 1
+	description "This sword may be a bit of a strange license, but its intention is to inspire pride to the wearer, and when the Unfettered see it they show respect. This was given by one of their warlords as a sign of trust, and it is quite handy in fights."
+	description "	Somehow, the blade feels very light, and as though it was accompanying the strikes of its owner. The blade itself is mostly intended to be defensive, and fits you quite well even though it was clearly meant for a Hai."
```

Comment:
```suggestion
	description "	When striking, somehow this blade feels very light, as though it was moving along with its wielder. The blade itself is mostly intended to be defensive and fits you quite well, despite clearly being meant for a Hai."
```

No need to expand it into needing the comma instead of just dropping the incorrect usage. Added benefit is the reduction in commas.

Suggestion for both general clean up of first sentence to be more precise on the way it feels oddly light, but also more ambiguous about it's possible intelligence by not referring to it as owned.